### DISCUSSION TITLE
IDE complains with "cannot infer argument" when using any CommandManager type

### DISCUSSION THREAD

**Discussion by Andre601** [Incendo/cloud#236] [Q&A] [answered]
When using any type of Command manager does my IDE complain with "cannot infer argument" within the diamond section (`<>`)

This is the code I use:
```java
    private void setupCommandFramework(){
        // I don't use a special command sender so I provide CommandSender and Function.identity()
        PaperCommandManager<CommandSender> manager = new PaperCommandManager<>(
                this,
                AsynchronousCommandExecutionCoordinator.newBuilder().build(),
                Function.identity(),
                Function.identity()
        );
    }
```
The error appears at the `<>` of `new PaperCommandManager` and confuses me a lot since this is exactly what [the docs tell you to do](https://incendo.github.io/cloud/#_command_manager)

**Comment by jpenilla** [comment, marked as answer]
Yeah, the docs have a mistake here, been meaning to fix it but haven't had time. The solution is giving a type argument to `newBuilder`, i.e. `AsynchronousCommandExecutionCoordinator.<CommandSender>newBuilder()`
